packages/now-next/package.json

```diff
@@ -29,7 +29,7 @@
     "@vercel/nft": "0.9.2",
     "async-sema": "3.0.1",
     "buffer-crc32": "0.2.13",
-    "escape-string-regexp": "3.0.0",
+    "escape-string-regexp": "2.0.0",
```

Comment:
Why the older version?

Reply:
We need to match the version Next.js is using or else replacing won't work correctly, added a comment [here](https://github.com/vercel/vercel/pull/5298/files#diff-d1fee2ddc0872d679e1cf35b915460f018264a315f528b2cca0a1923b854bf05R22)